package/fedora/freecad.spec

```diff
@@ -1,111 +1,99 @@
+# This package depends on automagic byte compilation
+# https://fedoraproject.org/wiki/Changes/No_more_automagic_Python_bytecompilation_phase_2
+%global _python_bytecompile_extra 1
+
+# Setup python target for shiboken so the right cmake file is imported.
+%global py_suffix %(%{__python3} -c "import sysconfig; print(sysconfig.get_config_var('SOABI'))")
 
 # Maintainers:  keep this list of plugins up to date
-# List plugins in %%{_libdir}/%{name}/lib, less '.so' and 'Gui.so', here
-%global plugins Drawing Fem FreeCAD Image Import Inspection Mesh MeshPart Part Points QtUnit Raytracing ReverseEngineering Robot Sketcher Start Web PartDesignGui _PartDesign Path PathGui Spreadsheet SpreadsheetGui area DraftUtils DraftUtils libDriver libDriverDAT libDriverSTL libDriverUNV libMEFISTO2 libSMDS libSMESH libSMESHDS libStdMeshers Measure TechDraw TechDrawGui libarea-native Surface SurfaceGui PathSimulator
+# List plugins in %%{_libdir}/freecad/lib, less '.so' and 'Gui.so', here
+%global plugins Complete DraftUtils Drawing Fem FreeCAD Image Import Inspection Mesh MeshPart Part PartDesign Path Points QtUnit Raytracing ReverseEngineering Robot Sketcher Spreadsheet Start Web PartDesignGui _PartDesign Spreadsheet SpreadsheetGui area
+
+# Some configuration options for other environments
+# rpmbuild --with=occ:  Compile using OpenCASCADE instead of OCE
+%global occ %{?_with_occ: 1} %{?!_with_occ: 0}
+# rpmbuild --with=bundled_zipios:  use bundled version of zipios++
+%global bundled_zipios %{?_with_bundled_zipios: 1} %{?!_with_bundled_zipios: 0}
+# rpmbuild --without=bundled_pycxx:  don't use bundled version of pycxx
+%global bundled_pycxx %{?_with_bundled_pycxx: 0} %{?!_with_bundled_pycxx: 1}
+# rpmbuild --without=bundled_smesh:  don't use bundled version of Salome's Mesh
+%global bundled_smesh %{?_with_bundled_smesh: 0} %{?!_with_bundled_smesh: 1}
 
-# Some plugins go in the Mod folder instead of lib. Deal with those here:
-%global mod_plugins Mod/PartDesign
 %define name freecad
 %define github_name FreeCAD
 %define branch master
 
 Name:           %{name}
 Epoch:          1
-Version:    	0.19_pre
+Version:        0.19_pre
 Release:        {{{ git_commits_no }}}
 Summary:        A general purpose 3D CAD modeler
 Group:          Applications/Engineering
 
 License:        GPLv2+
```

Comment:
Is this correct? FreeCAD is licensed LGPL 2, not GPL 2.

Reply:
Hmm... thanks for the catch. I took it from the official Fedora spec. I'll ask

Reply:
Spec discussion & fixes: https://www.freecadweb.org/tracker/view.php?id=4132
I'll submit another PR